compiler/rustc_span/src/hygiene.rs

```diff
@@ -982,6 +982,11 @@ pub struct ExpnData {
     /// Should debuginfo for the macro be collapsed to the outermost expansion site (in other
     /// words, was the macro definition annotated with `#[collapse_debuginfo]`)?
     pub(crate) collapse_debuginfo: bool,
+    /// Whether this macro expansion came from a compiler implemented macro (`rustc_builtin_macro`)
+    /// or a macro by example provided by the standard library (marked with
+    /// `rustc_diagnostic_item`). If this is true, then we do not display the standard note telling
+    /// people to use the `-Zmacro-backtrace` flag.
+    pub builtin_macro: bool,
```

Comment:
```suggestion
    pub hide_backtrace: bool,
```
(Or `suggest_backtrace`.)